phalcon/Url.zep

```diff
@@ -16,6 +16,7 @@ use Phalcon\Url\Exception;
 use Phalcon\Mvc\RouterInterface;
 use Phalcon\Mvc\Router\RouteInterface;
 use Phalcon\Di\InjectionAwareInterface;
+use Phalcon\Helper\Str;
```

Comment:
Don't think we need this one if we use preg_replace

Reply:
You are right.